### PR TITLE
Meeting notes 2020/01/28

### PR DESCRIPTION
**Pull request by jameslibscie**
I'm not the GIFmaster that Chris is. Sorry for the boring PR ;)

~~Also, I can't figure out how to make sure these commits get squashed. Tips are very much welcome.~~
EDIT: Nvm, I see it listed below now. Didn't have the option on a different PR, so got confused.

### CODE REVIEW

File: 20200128.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+        - Created a small example where I can test/play with this before moving to the sdk
+    - Questions
+        - James: Does the SDK distinguish between different ways of editing metadata in any way? Or is it just a json overwrite of the metadata that gets validated? Walk through roadmap feature list...
+            - Answer: Yes
+        - Chris: Roadmap?
+            - Answer: If 0.4 works, work can proceed to 0.5 with delete functionality
+-   [`@hypergraph-xyz/cli`](https://github.com/hypergraph-xyz/cli)
+    - Update by Julian
+        - Updating to SDK 0.4.0 is a WIP, Diego is debugging an issue
+            - Found through test, so easy to verify when it's fixed
```
Comment (juliangruber) [inline 20200128.md]:
```suggestion
            - Found through the cli's test suite, so easy to verify when it's fixed
```

File: 20200128.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+            - Answer: CLI first is preferred. CLI should be fully-featured, power user tool
+-   [`@hypergraph-xyz/desktop`](https://github.com/hypergraph-xyz/desktop)
+    - Update by Patch
+        - Creating a clickthrough from the flow and features James and I created.
+        - Will be finished and shared internally this week.
+        - Should be shown to 1-2 outside testers after several quick internal iterations.
+        - We need to discuss and formulate a plan/script for testing with people who are not immediately familiar with our project and product
+            - @jameslibsie should plan 
+    - Update by Julian
+        - Nothing
```
Comment (juliangruber) [inline 20200128.md]: can we omit these if there's nothing, to reduce noise?

File: 20200128.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+    - Questions
+-   [`@hypergraph-xyz/vault`](https://github.com/hypergraph-xyz/vault)
+    - Update by Julian
+        - Removed all mentions of the domain, as legalities need to be figured out still
+        - CC is requested upon first payment, rather than on registration
+        - Create basic API in preparation for the Worker
+        - Create basic Worker that will fetch modules from the Vault's API and replicate them (currently without any backend like s3)
+        - WIP: Deploying the worker to AWS, need to read up on more AWS resources. If this takes too long we might seek a contractor for the initial setup.
```
Comment (juliangruber) [inline 20200128.md]: the stripe WIP is missing, I updated the message in Slack right after we talked 😅

File: 20200128.md (hunk 2 above)
Comment (jameslibscie) [reply 20200128.md]: Done!

File: 20200128.md (hunk 3 above)
Comment (jameslibscie) [reply 20200128.md]: Added!

File: 20200128.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+        - Put together feature list for MVP with Chris & Patch. Basic [Roadmap](https://github.com/hypergraph-xyz/desktop/wiki/Roadmap) now live on the [Hypergraph Desktop wiki](https://github.com/hypergraph-xyz/desktop/wiki)
+            - Scoped in: Ephemeral project management tool for solo projects
+            - Scoped out: Vault, all sharing (networking) and collaboration
+            - Workflow: design - clickthrough test - iterate design - potential 2nd clickthrough test - code
+        - Planning 1-on-1s with the people I haven't met IRL yet to discuss how we can work together most effectively
+        - Next steps:
+            - Convert feature list to issues and create GitHub Projects (or similar) board(s) for managing the work
+            - Identify SDK, CLI and non-technical issues for work that needs doing prior to these identified Desktop issues
+- Roundabout
+    - Everybody gets a Chebleader
```
Comment (chartgerink) [inline 20200128.md]: Although I like inside jokes this can be confusing and clique forming 🤷‍♀

Comment (jameslibscie) [reply 20200128.md]: Noted for next time :)

**Pull request merged**